libr/bin/bin.c

```diff
@@ -379,8 +385,13 @@ static int string_scan_range(RList *list, const ut8 *buf, int min,
 				new->string = r_str_ndup ((const char *)tmp, i);
 				r_list_append (list, new);
 			} else {
-				// DUMP TO STDOUT. raw dumping for rabin2 -zzz
-				printf ("0x%08" PFMT64x " %s\n", str_start, tmp);
+				if (rdump) {
+					// DUMP the strings to r2 shell for izzz
+					r_cons_printf ("0x%08" PFMT64x " %s\n", str_start, tmp);
```

Comment:
use io->cb_printf, instead of including rcons here, we cant make rbin depend on rcons